Core/src/main/java/com/github/intellectualsites/plotsquared/plot/PlotSquared.java

```diff
@@ -1525,6 +1558,44 @@ public void copyFile(String file, String folder) {
     public void disable() {
         try {
             eventDispatcher.unregisterAll();
+            if (HybridUtils.UPDATE && Settings.Enabled_Components.PERSISTENT_ROAD_REGEN && (
+                !HybridUtils.regions.isEmpty() || !HybridUtils.chunks.isEmpty())) {
+                PlotSquared.log(Captions.PREFIX
+                    + "Road regeneration incomplete. Saving incomplete regions to disk.");
+                PlotSquared.debug("   Regions: " + HybridUtils.regions.size());
+                PlotSquared.debug("   Chunks: " + HybridUtils.chunks.size());
+                try {
+                    ArrayList<Integer[]> regions = new ArrayList<>();
```

Comment:
I don't know how much data is serialized here, but using `ArrayList<int[]>` instead of `ArrayList<Integer[]>` will likely decrease the file size significantly

Reply:
ab9e84d4718b8c567b59b69d7d09b673c99e1689